src/Utility/ExprFunctions.C

```diff
@@ -504,19 +489,6 @@ void STensorFuncExpr::setNoDims ()
 }
 
 
-template<>
-SymmTensor STensorFuncExpr::evaluate (const Vec3& X) const
-{
-  SymmTensor sigma(nsd,p.size()==4);
```

Comment:
Can't see how you account for the second bool argument (equals p.size()==4) in the general implementation for the SymmTensor case. Will it work? (Or is it handled in later commits, perhaps).

Reply:
that's fully taken into account. added tests to confirm.

Reply:
Indeed, I oversaw the fact you are here using the (std::vector) constructor and not the (int,bool) one.